TestON/tests/IntentPerfNext/IntentPerfNext.py

```diff
@@ -37,7 +37,8 @@ def CASE1( self, main ):
         main.ONOSbench.onosUninstall( nodeIp=ONOS4Ip )
         main.ONOSbench.onosUninstall( nodeIp=ONOS5Ip )
         main.ONOSbench.onosUninstall( nodeIp=ONOS6Ip )
```

Comment:
The commits are from andrew, so either you were both on the same machine,
or somehow you pulled them into your branch.

On Wed, Feb 18, 2015 at 8:04 PM, hakrishn <notifications@github.com> wrote:

> In TestON/tests/IntentPerfNext/IntentPerfNext.py
> <https://github.com/opennetworkinglab/ONLabTest/pull/63#discussion_r24964796>
> :
>
> > @@ -37,7 +37,8 @@ def CASE1( self, main ):
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS4Ip )
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS5Ip )
> >          main.ONOSbench.onosUninstall( nodeIp=ONOS6Ip )
>
> I did not change any driver files after pulling Kelvins mininet changes
> sometime back. So, this commit should have only
> OnosCHO file changes.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opennetworkinglab/ONLabTest/pull/63/files#r24964796>.
>



-- 
Jon Hall
ON.Lab
jhall@onlab.us